util/coordinator.py

```diff
@@ -0,0 +1,706 @@
+from __future__ import absolute_import, division, print_function
```

Comment:
It's good to have this in an extra file, as this should go away soon. We should soon move to an in-graph replica scheme for training.